dists/redhat/scummvm.spec

```diff
@@ -92,7 +92,7 @@ fi
 %{_datadir}/pixmaps/scummvm.xpm
 %{_datadir}/icons/hicolor/48x48/apps/scummvm.png
```

Comment:
We should consider renaming the icon files as well (directly in _dists_, not in the _redhat_ subdirectory) to match its desktop counterpart names. This is optional, but would be helpful for the ScummVM Flatpak.

Reply:
I didn't modify icons located in icons folder because they could be used by other platforms and this id thing is quite FreeDesktop specific.
But I moved them when installing them and adapted the path in desktop and metainfo.